lib/sycamore/sycamore/data/table.py

```diff
@@ -305,6 +308,7 @@ def to_pandas(self) -> DataFrame:
             index=None,
             columns=flattened_header if max_header_prefix_row >= 0 else None,
         )
+        self.column_headers = flattened_header if max_header_prefix_row >= 0 else []
```

Comment:
I really don't like mutating the table object in this method. Can we avoid this or do it in like init or something?